NewArch/src/App.tsx

```diff
@@ -276,28 +281,60 @@ const DrawerListView = (props) => {
   );
 };
 
-function CustomDrawerContent({navigation}) {
+function CustomDrawerContent({ navigation }: { navigation: any }) {
   const isDrawerOpen =
     getDrawerStatusFromState(navigation.getState()) === 'open';
 
   const navigationState = navigation.getState();
   const currentRoute = navigationState.routeNames[navigationState.index];
 
-  if (!isDrawerOpen) {
-    return <View />;
-  }
+  // Refs for hamburger, home and settings buttons
+  const hamburgerRef = useRef<Pressable>(null);
+  const homeRef = useRef<Pressable>(null);
+  const settingsRef = useRef<Pressable>(null);
+
+  // When drawer opens, focus the Home menu item
+  useEffect(() => {
+    if (isDrawerOpen && homeRef.current) {
+      setTimeout(() => {
```

Comment:
Requiring a setTimeout is an indication of a synchronization issue.  And the 300ms is arbitrary. 
If we have a ref to home, I think it should already be mounted, and we should be able to call focus on it?